.github/workflows/build_wamr_vscode_ext.yml

```
@@ -0,0 +1,60 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+name: build wamr-ide vscode extension
+
+on:
+  workflow_call:
+    inputs:
+      upload_url:
+        description: a semantic version number. it is required when `release` is true.
```

In this case, there is no `release` as a argument.
```suggestion
        description: a semantic version number. 
```